plugin/pkg/admission/storage/storageobjectinuseprotection/admission.go

```diff
@@ -119,3 +128,25 @@ func (c *storageProtectionPlugin) admitPVC(a admission.Attributes) error {
 	pvc.Finalizers = append(pvc.Finalizers, volumeutil.PVCProtectionFinalizer)
 	return nil
 }
+
+func (c *storageProtectionPlugin) admitVAC(a admission.Attributes) error {
```

Comment:
please add unit test for these changes.

Reply:
Please add a release note for this PR. Because it introduces a user-facing change.